pkg/descheduler/controllers/migration/arbitrator/arbitrator.go

```diff
@@ -102,13 +111,38 @@ func (a *arbitratorImpl) Add(job *v1alpha1.PodMigrationJob) {
 	a.waitingCollection[job.UID] = job.DeepCopy()
 }
 
+// Delete remove PodMigrationJob from local PassedArbitration Record map.
+func (a *arbitratorImpl) Delete(uid types.UID) {
+	a.filter.removeJobPassedArbitration(uid)
+}
+
 // Start starts the goroutine to arbitrate jobs periodically.
 func (a *arbitratorImpl) Start(ctx context.Context) error {
 	klog.InfoS("Start Arbitrator Arbitrate Goroutine")
 	wait.Until(a.doOnceArbitrate, a.interval, ctx.Done())
 	return nil
 }
 
+// Filter checks if a pod can be evicted
+func (a *arbitratorImpl) Filter(pod *corev1.Pod) bool {
+	if !a.filter.filterExistingPodMigrationJob(pod) {
+		return false
+	}
+
+	if !a.filter.reservationFilter(pod) {
+		return false
+	}
+	return true
```

Comment:
Here, we should rollback original implementation.  Other native plugins call the method to determine whether can create new eviction/pmj, so we should check the non-retryable filter and retryable filter.

Reply:
ok